machine-learning/ml_tips/README.md

```diff
@@ -0,0 +1,75 @@
+# ML tips workflow
+
+This workflow aims to explain common ML techniques using Hivemall and Treasure Workflow.
+
+Using [titanic data](https://github.com/amueller/scipy-2017-sklearn/blob/master/notebooks/datasets/titanic3.csv), this workflow explains the following basic techniques.
+
+- Random/stratified splitting for train and test data
+- Feature normalization
+- Missing value imputation
+
+## Input
+
+This workflow assume a table as follows:
+
+|pclass<br/>`int`|survived<br/>`int`|name<br/>`string`|sex<br/>`string`|age<br/>`int`|sibsp<br/>`int`|parch<br/>`int`|ticket<br/>`string`|fare<br/>`double`|cabin<br/>`string`|embarked<br/>`string`|boat<br/>`string`|body<br/>`int`|home.dest<br/>`string`|
+|---------|------|---------|--------|-------|-------|-------|--------|-------|-------|-----------|--------|---------|-------|
+|1|1|"Allen, Miss. Elisabeth Walton"|female|29|0|0|24160|211.3375|B5|S|2||"St Louis, MO"|
+|1|1|"Allison, Master. Hudson Trevor"|male|0.9167|1|2|113781|151.5500|C22 C26|S|11||"Montreal, PQ / Chesterville, ON"|
+|1|0|"Allison, Miss. Helen Loraine"|female|2|1|2|113781|151.5500|C22 C26|S|||"Montreal, PQ / Chesterville, ON"|
+|...|...|...|...|...|...|...|...|...|...|...|...|...|...|
+
+Here is a description of each column.
+
+    pclass          Passenger Class
+                    (1 = 1st; 2 = 2nd; 3 = 3rd)
+    survival        Survival
+                    (0 = No; 1 = Yes)
+    name            Name
+    sex             Sex
+    age             Age
+    sibsp           Number of Siblings/Spouses Aboard
+    parch           Number of Parents/Children Aboard
+    ticket          Ticket Number
+    fare            Passenger Fare
+    cabin           Cabin
+    embarked        Port of Embarkation
+                    (C = Cherbourg; Q = Queenstown; S = Southampton)
+    boat            Lifeboat
+    body            Body Identification Number
+    home.dest       Home/Destination
+
+In this workflow, you will predict the `survival` column as a binary classification problem.
+
+## Workflow
+
+```sh
+$ ./data.sh # Upload sample dataset to Treasure Data
+$ td wf push ml_tips
+$ td wf start ml_tips titanic --session now
+```
+
+- [titanic.dig](titanic.dig) - TD workflow script for linear regression
+- [titanic_pipeline.yml](titanic_pipeline.yml) - Source file of dig file and queries for [molehill](https://github.com/chezou/molehill)
```

Comment:
I personally believe unofficial, third-party dependency shouldn't be preferred because TD customers and internal support team will be confused. Why don't you delete the line and YAML file until `molehill` becomes GA?

Reply:
better to kept in `[WIP] PR` until molehill released a stable release.

Reply:
OK, will delete YAML file description  until molehill becomes GA. I think this PR itself is an ordinal workflow, so we can discuss on it.

Reply:
Removed both description and YAML file c2379ec 491e070